DependencyInjection/Configuration.php

```diff
@@ -388,6 +388,7 @@ private function addExceptionSection(ArrayNodeDefinition $rootNode)
                             ->useAttributeAsKey('name')
                             ->prototype('boolean')->end()
                         ->end()
+                        ->booleanNode('debug')->treatNullLike(false)->end()
```

Comment:
I think so as it enables us to avoid the inconsistencies we would need to discuss otherwise here.